modules/doc/content/getting_started/new_users.md.template

```diff
@@ -2,6 +2,11 @@
 
 Welcome to MOOSE! This page describes some helpful ways to start developing with MOOSE.
 
+!alert! note
+If you arrived here directly, be sure to have completed one of the "Getting Started" instruction
```

Comment:
```suggestion
If you arrived here directly, be sure to complete one of the "Getting Started" instruction
```

Reply:
I'll gladly make this change! I was trying to make it more "past-tense" so as to not make the user go through it again. Can you do the following, and tell me still that what you want is okay?

Go here: https://mooseframework.inl.gov/docs/PRs/25728/site

Then click on "Getting Started" and then "Linux and MacOS". And at the bottom right of the page, click the "New Users" button.

I felt by having a 'future-tense' like statement, they might wind up trying to run through the installation again?

I know it is a simple change, this PR. But this simple wrap-around behavior actually stumped me how to write it in such a way as to prevent a new user from becoming confused.

Reply:
Sorry for the time in getting back to this - I forgot about it! The phrase "to have completed" is actually considered a *future perfect* tense (essentially it implies projecting our perspective ahead in time to the future and looking back to a particular event that was completed). The phrase "to complete" here is more present and active.